src/main/java/net/minecraftforge/common/Tags.java

```diff
@@ -180,6 +182,19 @@ private static TagKey<Block> tag(String name)
         }
     }
 
+    public static class EntityTypes
+    {
+        private static void init() {}
+
+        public static final TagKey<EntityType<?>> BOSSES = tag("bosses");
+
+        private static TagKey<EntityType<?>> tag(String name)
+        {
+            return TagKey.create(Registry.ENTITY_TYPE_REGISTRY, new ResourceLocation("forge", name));
+        }
+    }
+
```

Comment:
Formatting: there should only be one blank line, not two.

Reply:
fixed!